### PR TITLE
Prime.py

### PR DESCRIPTION
**Pull request by Priyanshu729**
Surely amazing

### CODE REVIEW

**Comment by tasnimzotder** [comment]
hey @Priyanshu729 there's a conflicting file `python/palindrome_checker.py`. You need to remove the file from your commit to resolve the conflict. You can do this by the following commands.

1. Remove the last 4 commits
```bash
git reset --soft HEAD~4
```

2. Reset the file `python/palindrome_checker.py`
```bash
git reset python/palindrome_checker.py
```

3. Commit the changes
```bash
git commit -m " ... MESSAGE ... "
```

4. Push the changes
```bash
git push -f origin master
```

File: compute.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import tkinter
```
Comment (tasnimzotder) [inline compute.py:1]: Codacy found an issue: ['tkinter' imported but unused (F401)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- compute.py  5
- python/binary_search.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655&bid=20438245)

File: python/binary_search.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+def binarysearch(sp,ep,l1,item):
+    for i in range(sp,ep):
```
Comment (tasnimzotder) [inline python/binary_search.py:2]: Codacy found an issue: [Unused variable 'i'](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655)

File: python/binary_search.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+element=int(input())
+e_p=len(l)-1
+s_p=0
+f=binarysearch(s_p,e_p,l,element)
+if f!=0:
+    print("ELEMENT IS FOUND AT",f)
+else:
+    print("ELEMENT IS NOT FOUND")
+
+
```
Comment (tasnimzotder) [inline python/binary_search.py:22]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655)

File: python/binary_search.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+def binarysearch(sp,ep,l1,item):
+    for i in range(sp,ep):
+        mid=(sp+ep)//2
+        if l1[mid]==item:
+            return mid
+        elif item>l1[mid]:
+            sp=mid+1
+        elif item<l1[mid]:
+            ep=mid-1
+    else:
```
Comment (tasnimzotder) [inline python/binary_search.py:10]: Codacy found an issue: [Else clause on loop without a break statement](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655)

File: python/binary_search.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        mid=(sp+ep)//2
+        if l1[mid]==item:
+            return mid
+        elif item>l1[mid]:
+            sp=mid+1
+        elif item<l1[mid]:
+            ep=mid-1
+    else:
+        return 0
+l=eval((input()))
```
Comment (tasnimzotder) [inline python/binary_search.py:12]: Codacy found an issue: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279655)

**Comment by Priyanshu729** [comment]
U can please solve my issue for this plssss

**Comment by Priyanshu729** [comment]
Pls anyone help me😭😭

**Comment by Priyanshu729** [comment]
Pls help me😭😭

**Pull request closed without merging**